oddbird/oddleventy
# Update accoutrement

![](https://source.unsplash.com/featured/?cute,animal&acct)






## Steps to test/reproduce
- ` yarn ` to update dependencies
- `yarn serve` to run the site
- make sure Accoutrement is working in all of the places where we are using it. 
- Make sure the styleguide is showing correctly

_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._



# REMEMBER: Attach this PR to the Trello card
